man/man8/zfs.8

```diff
@@ -2585,6 +2585,22 @@ Any property specified on the command line using the
 .Fl o
 option is ignored.
 If the target filesystem already exists, the operation completes successfully.
+.It Fl n
+Do a dry-run
+.Pq Qq No-op
+creation.
+No datasets will be created.
+This is useful in conjunction with the
+.Fl v
+or
+.Fl P
+flags to validate properties that are passed via
+.Fl o
+options and those implied by other options.
```

Comment:
Similar to the `zpool create` dry-run command I think it would be a good idea to include the following additional sentence.   "The actual dataset creation can still fail due to insufficient privileges or available capacity."

Reply:
ok